_posts/2017-12-30-JS-Conf-Budapest-day-2.md

```diff
@@ -0,0 +1,387 @@
+---
+layout: post
+authors: [frederic_ghijselinck, orjan_de_smet, stefanie_geldof, martijn_willekens, dimitri_de_kerf, dries_gijssels]
+title: 'JSConf Budapest 2017 Day 2'
+image: /img/js-conf-budapest-2017/header.png
+tags: [JS Conf Budapest,JavaScript,Conference]
+category: Conference
+comments: true
+---
+
+## From JSConf Budapest with love
+
+This year's edition of JS Conf Budapest returned to the first venue at [Urania National Movie theater](http://www.urania-nf.hu/).
+
+> Uránia Cinema in the middle of the city, near the party-district.
+> Designed by Henrik Schmahl in the late 1890's, the interior is characterized by the ornamental motifs of the Venetian Gothic and Moor styles.
+> The place is listed as the world's 3rd most beautiful cinema on Bored Panda.
+> Many tech conferences were hosted here recently, such as TEDx and Strech Conference, because of the unique atmosphere.
+
+JS Conf Budapest 2017 is hosted by [Glen Maddern](https://twitter.com/glenmaddern) and [Charlie Gleason](https://twitter.com/superhighfives).
+At 10:00 the second day of the conference started. Enough time to drink great coffee and enjoy the breakfast.
+
+****
+
+<img class="image fit" src="{{ '/img/js-conf-budapest-2017/js-conf-budapest-2017-collage.jpg' | prepend: site.baseurl }}" alt="JS Conf Budapest 2017 Photo Collage" />
+
+****
+
+## Day 2: Talks
+
+* [Don Burks: MVC - What a web app and a Mozart Violin Concerto have in common](#don-burks-mvc---what-a-web-app-and-a-mozart-violin-concerto-have-in-common)
+* [Opher Vishnia: Web Animation: from Disney to SASS](#opher-vishnia-web-animation-from-disney-to-sass)
+* [Imad Elyafi: Migrating Pinterest profiles to React](#imad-elyafi-migrating-pinterest-profiles-to-react)
+* [Laura Carvajal: YES, your site can (and should) be accessible too. Lessons learned in building FT.com](#laura-carvajal-yes-your-site-can-and-should-be-accessible-too-lessons-learned-in-building-ftcom)
+* [Nikita Baksalyar: Exploring the P2P world with WebRTC & JavaScript](#nikita-baksalyar-exploring-the-p2p-world-with-webrtc--javascript)
+* [Vaidehi Joshi: Goldilocks And The Three Code Reviews](#vaidehi-joshi-goldilocks-and-the-three-code-reviews)
+
+****
+
+## Day 2: Morning
+
+<span class="image left"><img class="p-image" alt="Don Burks" src="/img/js-conf-budapest-2017/speaker-don.jpg"></span>
+
+
+### Don Burks: MVC - What a web app and a Mozart Violin Concerto have in common
```

Comment:
Nice summary, but missing the point a bit (had to really look up the presentation). Maybe add the slide where devs and musicians are compared at the end? Nothing wrong with adding that to the blog post.